doc/modules/con_Approval.adoc

```diff
@@ -3,19 +3,21 @@
 [id="Approval"]
 = Working with approval
 
-Catalog supports multi-level approval workflows, providing the flexibility to create a tiered administrative process that includes selected organizational groups. Administrators can design approval workflows, view request details, approve or deny requests, and monitor status of existing requests. Once created, an approval workflow can be assigned to Catalog portfolios.
+Catalog supports group-based approval processes applied to portfolios, products, and platform inventories, providing the flexibility to include selected organizational groups in approving requests.
+
+Approval administrators can create approval groups and processes, view request details, approve or deny requests, and monitor status of existing requests.
```

Comment:
We need to include a note that the Approval Administrator also needs to be an Organization Admins on the Platform to create groups and list the groups when creating Approval processes.  

Organization Admins are defined at the SSO level, not in the Platform RBAC service.

Reply:
I've added a note in the lates commit. I think we'll need to add a separate doc explaining SSO level vs RBAC service, but not sure how Catalog specific this would be at this point?